Makefile

```diff
@@ -11,6 +11,7 @@ DOCKER_IMAGE_TAG  ?= $(subst /,-,$(shell git rev-parse --abbrev-ref HEAD))-$(she
 # but for simplicity sake we just make sure they exist in the first one, and
 # then keep using those.
 FIRST_GOPATH      ?= $(firstword $(subst :, ,$(shell go env GOPATH)))
+DEP_GOPATH        ?= /tmp/go
```

Comment:
What about a more unique place like `/tmp/thanos-go`, plus let's name it `TMP_GOPATH` maybe? To name what is it actually.